README.md

```diff
@@ -513,7 +369,7 @@ customer_ingress:
     alb.ingress.kubernetes.io/certificate-arn: <CERT_ARN>
     alb.ingress.kubernetes.io/healthcheck-path: /health
     alb.ingress.kubernetes.io/healthcheck-protocol: HTTP
-    alb.ingress.kubernetes.io/listen-ports: '[{"HTTPS": 5039}]'
+    alb.ingress.kubernetes.io/listen-ports: '[{"HTTPS": 5039}, {"HTTPS": 3030}]'
```

Comment:
This seems incorrect? The ALB for Sombra does not listen for the Pathfinder service too, right?

Reply:
Will defer to @shashiranjan84 on that.

I reordered the examples, so it looks diffed, but I didn't actually change the config